### ISSUE TITLE
Getting a BadMethodCallException on PHP Artisan on Laravel 5

### ISSUE THREAD

**Issue by GumptionTechnologies** [KnightSwarm/laravel-saml#18]
Hi,

Thanks for writing a good package for SAML but I need to know if the package is good for Laravel 5 too?

I am getting BadMethodCalException on PHP Artisan when I run this "php artisan config:publish knight-swarm/laravel-saml"

"call to undefined method [package]"

**Comment by cby016** [comment]
**GumptionTechnologies** were you able to get this installed on laravel 5.2? If so how did you get around the "[BadMethodCallException]  Call to undefined method [package]" error?

**Comment by GumptionTechnologies** [comment]
I wasn't able to get it going. However, I made direct calls to my SAML classes and got it completed. If you need that way, I can help you on that.

**Comment by cby016** [comment]
Yeah if you could show me how you did it that would be great. Did you use simplesamlphp or did you create something totally custom?

**Comment by GumptionTechnologies** [comment]
I used simplesamlphp and got it working within Laravel 5.2. Can we discuss your problem on Skype?

**Comment by cby016** [comment]
**GumptionTechnologies** thanks for the willingness to help out. I actually got it working with the https://github.com/aacotroneo/laravel-saml2 library.

**Comment by GumptionTechnologies** [comment]
Welcome my friend. But this package doesn't help with SimpleSAMLPHP. Right?

**Comment by cby016** [comment]
No it uses onelogin/php-saml.

**Comment by kannan567** [comment]
@GumptionTechnologies Could you please share me the details how to integrate the  simplesamlphp in laravel.
